module/VuFind/src/VuFind/ILS/Driver/Koha.php

```diff
@@ -402,6 +427,50 @@ public function getMyTransactions($patron)
     }
 
     /**
+     * Check whether the patron has any blocks on their account.
+     *
+     * @param array $patron Patron data from patronLogin
+     *
+     * @throws ILSException
+     *
+     * @return mixed A boolean false if no blocks are in place and an array
+     * of block reasons if blocks are in place
+     */
+    public function getAccountBlocks($patron)
+    {
+        $blocks = [];
+
+        try {
+            $id = $patron['id'];
+            $sql = "select type as TYPE, comment as COMMENT " .
+                "from borrower_debarments " .
+                "where (expiration is null or expiration >= NOW()) " .
+                "and borrowernumber = :id";
+            $sqlStmt = $this->db->prepare($sql);
+            $sqlStmt->execute([':id' => $id]);
+
+            foreach ($sqlStmt->fetchAll() as $row) {
+                $block = empty($this->blockTerms[$row['TYPE']])
```

Comment:
I found this line a bit hard to read as previously formatted. I realize that my proposed change isn't perfect either. Please take a look and let me know if I broke anything or if you object to this revision.

Reply:
(The way GitHub has highlighted this makes it hard to see what I'm talking about -- I'm referring to the ternary operation to initialize $block to an array containing the appropriate block description. I refactored it both to incorporate an empty check (in case of previously undefined block types) and to make it more clear that we are creating an array, since I found that wrapping the whole ternary operation inside `[ ... ]` was a bit harder to understand.

Reply:
I must have written that line 3 or 4 different ways to try, however I suspect your way will be more consistent with the rest of the codebase. So more than happy to go with it.

(I've run the new version against my dev environment and it worked fine)